clients/src/main/java/org/oracle/okafka/clients/admin/KafkaAdminClient.java

```diff
@@ -1680,32 +1700,24 @@ void handleFailure(Throwable throwable) {
 	}
 
 	@Override
-	public DeleteTopicsResult deleteTopics(Collection<String> topicNames, DeleteTopicsOptions options) {
-		org.oracle.okafka.clients.admin.DeleteTopicsResult delResult = deleteTopics(topicNames,
-				new org.oracle.okafka.clients.admin.DeleteTopicsOptions());
-		KafkaFuture<Void> ftr = delResult.all();
-		try {
-			ftr.get();
-		} catch (Exception e) {
-			log.error("Exception from deleteTopics", e);
+	public DeleteTopicsResult deleteTopics(final TopicCollection topics, DeleteTopicsOptions options) {
+
+		if (topics instanceof TopicIdCollection) {
+			throw new FeatureNotSupportedException("Topic Id's are not supported for deleteTopics request");
```

Comment:
Topic Ids are  not supported for this release of OKafka.

Reply:
Okay.

Reply:
I changed it in this same pull request.